script/install.sh

```diff
@@ -67,6 +77,15 @@ set +v
 ######################################
 ## End
 
+
+######################################
+## Set back to https if needed
+if [[ $FORCE_SSH == "true" ]]
```

Comment:
Should be false

Reply:
Si on a set à ssh, je remet à https pour éviter les conflits avec git

Reply:
comme ça, le .rosinstal utilise toujours https mais est changé au besoin. Un peut comme du JIT/JAT